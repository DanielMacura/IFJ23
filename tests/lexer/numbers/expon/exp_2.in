let x = 1.23e-12