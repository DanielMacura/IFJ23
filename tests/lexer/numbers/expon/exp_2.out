KW_LET
VARIABLE
EQUALS
INTEGER:1.23e-12
EOF
